docs/resources/ocean_aks_np.md

```diff
@@ -276,3 +276,35 @@ The following arguments are supported:
     * `min_nics` - (Optional) Minimum number of network interfaces.
     * `min_disk` - (Optional) Minimum number of data disks available.
     * `vm_types` - (Optional, Enum `"generalPurpose", "memoryOptimized", "computeOptimized", "highPerformanceCompute", "storageOptimized", "GPU"`) The filtered vm types will belong to one of the vm types from this list.
+
+<a id="update-policy"></a>
+## Update Policy
+
+* `update_policy` - (Optional)
+  * `should_roll` - (Required) Enables the roll.
+  * `conditioned_roll` - (Optional, Default: false) Spot will perform a cluster Roll in accordance with a relevant modification of the cluster’s settings. When set to true , only specific changes in the cluster’s configuration will trigger a cluster roll (such as AMI, Key Pair, user data, instance types, load balancers, etc).
+  * `conditioned_roll_params` - (Optional) A custom list of attributes will trigger the cluster roll operation (overrides the predefined list of parameters). Valid only when the `conditioned_roll` parameter is set to true. (Valid values: `"availability_zones"`, `"max_pods_per_node"`, `"enable_node_public_ip"`, `"os_disk_size_gb"`, `"os_disk_type"`, `"os_sku"`, `"kubernetes_version"`, `"vnet_subnet_ids"`, `"pod_subnet_ids"`, `"labels"`, `"taints"`, `"tags"`, `"filters"`)
+  * `roll_config` - (Required) While used, you can control whether the group should perform a deployment after an update to the configuration.
+    * `batch_size_percentage` - (Required) Value as a percent to set the size of a batch in a roll. Valid values are 0-100. In case of null as value, the default value in the backend will be 20%.
```

Comment:
In case of null as value, the default value in the backend will be 20%.

Need confirmation on the above line